.bcr/patches/rust_dev_dep.patch

```diff
@@ -0,0 +1,36 @@
+diff --git a/MODULE.bazel b/MODULE.bazel
```

Comment:
Does this have to live here? Or can it live in the name spaced third_party folder?

Reply:
it has to be here for the publish-to-bcr app to locate it and apply the patch to the MODULE file that gets published to the registry.